CHANGELOG.md

```diff
@@ -2998,3 +2998,9 @@ This is a full list of proofs that have changed form to use irrelevant instance
   ```agda
   <-weakInduction-startingFrom : P i →  (∀ j → P (inject₁ j) → P (suc j)) → ∀ {j} → j ≥ i → P j
   ```
+
+* Added new functions to `Data.Vec.Relation.Binary.Equality.Setoid`
+  ```agda
+  cong-[]≔ : i p → xs ≋ ys → xs [ i ]≔ p ≋ ys [ i ]≔ p
+  map-[]≔ : i (f : B → A) (xs : Vec B n) p → map f xs [ i ]≔ f p ≋ map f (xs [ i ]≔ p)
```

Comment:
Ditto here: no real need for any of `i`, `f`, `xs` or `p`.

But there's a separate issue: the ambient convention that equational/unfolding laws be stated in the form 
<more complex expression> goes to <simpler expression>
suggest that this should perhaps better be
```agda
map f (xs [ i ]≔ p)  ≋ map f xs [ i ]≔ f p
```
but I'd check this with @MatthewDaggitt or @gallais who know the conventions better than I do!

Reply:
I changed the order of this.